ts/helpers/attributesHandler.ts

```diff
@@ -0,0 +1,29 @@
+
+export const getAttributeByPath = (slideAttributes, path) => {
+
+    if(!Array.isArray(path)){
+        throw Error("Invalid path");
+    }
+
+    if(slideAttributes === undefined) {
+        return undefined;
+    }
+
+    for(const node of path){
+        slideAttributes = slideAttributes[0][node];
+        if (slideAttributes === undefined) {
+            return [];
+        }
+    }
+
+    return slideAttributes;
+}
+
+export const cleanupJson = (element) => {
```

Comment:
this can go into a utils files no?

Reply:
created a utils folder and common.ts, moved it there.
let me know if you prefer any other structure or naming.

Reply:
it's fine for now, if we end up adding too many things to common.ts then we can consider moving this function out to json.ts